### DISCUSSION TITLE
Add assorted rare trinkets from Tbc Normal/Heroic dungeons to the Mythic Vendor List

### DISCUSSION THREAD

**Discussion by QuantaCosta** [chromiecraft/chromiecraft#6024] [Suggestions]
As mentioned above, I suggest to add 3 trinkets to the Mythic Token Vendor, namely:

1. - Quaggmirran's Eye (Slave Pens Heroic)
2. - Hourglass of the Unraveller (Black Morass Normal & Heroic)
3. - Scarab of the Infinite Cycle (Black Morass Normal & Heroic)

**Comment by balleny** [comment]
to some part I agree, but:
why only those 3? why not the Arcatraz caster trinket? why not the CoT1 caster trinket (hit + on-use spellpower). Why not the healer trinket from mana tombs (mp5 + on-use spellpower), why not the pvp-healer trinket from Underbog (resilience + mana proc), why not the several tank trinkets from Shadowlabs, Shattered halls, etc.?

But, why only trinkets? there are highly contested rings like the one from CoT1 for casters, too? Your selection seems arbitrary and not like a holistic solution.

**Comment by andovcr** [comment]
Could we just make it so that all Rare/Epic loot is available from the Mythic vendor?  If the reward of running Mythics is the bad-luck insurance, then there shouldn't be a philosophical argument against including Rare loot as well, correct?
